Dockerfile

```diff
@@ -0,0 +1,42 @@
+# created by Naga Ravi Chaitanya Elluri
+# Edited by Naga Ravi Chaitanya Elluri
+
+FROM ubuntu:latest
+MAINTAINER nelluri@deis.com
+RUN apt-get update
+RUN apt-get -y install make python lsb-release unzip wget vim git curl
+RUN wget https://storage.googleapis.com/kubernetes-release/release/v1.0.1/bin/linux/amd64/kubectl
+RUN chmod +x kubectl
+RUN mv kubectl /usr/local/bin/
+WORKDIR /home
+RUN wget --no-check-certificate https://dl.bintray.com/deis/helm/helm-v0.1.0+c600ce8-linux-amd64.zip
+RUN unzip helm-v0.1.0+c600ce8-linux-amd64.zip
+RUN mv helm /usr/local/bin/
+RUN apt-get install jq
```

Comment:
You should combine apt-get installs (see above and below).

The strategy should be:
- install all sources you'll need
- `apt-get update`
- `apt-get install <all your packages>`